src/elements/Toggle/Toggle.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Toggle
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Common from '../Common/Common';
+
+const Toggle = ({
+	checked,
+	value,
+	loading,
+	...other
+}) => {
+	const combinedClasses = classNames(
+		'ui-toggle',
+		other.classes,
+		{
+			on: checked,
+			loading
+		}
+	);
+
+	return (
+		<Common
+			{...other}
+			classes={combinedClasses}
+			tag="div"
+			data-checked={checked}
+			data-value={value}
```

Comment:
Once you change this to an `input`, you won't need these data attributes anymore

Reply:
removed data- attributes.